entity-framework/core/miscellaneous/connection-strings.md

```diff
@@ -93,3 +93,4 @@ public void ConfigureServices(IServiceCollection services)
         options.UseSqlServer(Configuration.GetConnectionString("BloggingDatabase")));
 }
 ````
+Note: the following using statement is required: using Microsoft.EntityFrameworkCore;
```

Comment:
> We're trying to eliminate [!NOTE] and the other obtrusive dialogs

@Rick-Anderson ah, interesting. Do you think the goal should be rising the bar for adding notes or warning and getting rid of excessive usage of notes (e.g. we are planning to get rid of the one we currently have on every page that says it the page applies only for EF Core and not EF6) or about removing the special formatting? I believe if we only had very few notes and warnings then it would be nice if to keep the special formatting.